plugins/itemsApi.ts

```diff
@@ -0,0 +1,26 @@
+import { Context } from '@nuxt/types';
+import { ItemsApi } from '../api/api';
+import { Configuration } from '../api/configuration';
+
+declare module '@nuxt/types' {
+  interface Context {
+    $ItemsApi: ItemsApi;
```

Comment:
Slight casing error here, your injected object would be `$itemsApi` and not `$ItemsApi`

Reply:
Fixed 👍